api/src/main/resources/org/openmrs/api/db/hibernate/Drug.hbm.xml

```diff
@@ -46,11 +46,14 @@
 			column="date_retired" not-null="false" length="19" />
 		<property name="retireReason" type="java.lang.String"
 			column="retire_reason" length="255" />
+		
 
 		<many-to-one name="concept" class="org.openmrs.Concept"
 			not-null="true" column="concept_id" />
 		<many-to-one name="dosageForm" class="org.openmrs.Concept"
 			not-null="false" column="dosage_form" />
+		<many-to-one name="doseLimitUnits" class="org.openmrs.Concept"
```

Comment:
Have you actually run these changes and confirmed that they work?

Reply:
am going to run them

Reply:
@dkayiwa i have run the changes and the work as seen here https://pastebin.com/ubevYdsX